src/api/senior/wishSeniorApply.ts

```
@@ -0,0 +1,29 @@
+export const wishSeniorApply = async (
+  seniorApplyReq: WishSeniorApplyRequest,
+) => {
+  try {
+    return (
+      await withOutAuthInstance.post<WishSeniorApplyResponse>(
+        '/wish/apply',
+        seniorApplyReq,
+      )
+    ).data;
+  } catch (e) {
+    throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)